app/app.go

```diff
@@ -34,9 +34,9 @@ import (
 	tmos "github.com/tendermint/tendermint/libs/os"
 	dbm "github.com/tendermint/tm-db"
 	"github.com/zigbee-alliance/distributed-compliance-ledger/docs"
+	dclcompltypes "github.com/zigbee-alliance/distributed-compliance-ledger/types/compliance"
```

Comment:
Do we really need to move ALL compliance types to `distributed-compliance-ledger/types/compliance`? It seems it's enough to move `compliance_info` only (as this is the type causing loop dependencies).

Reply:
You are right. Moved types back